src/button.tsx

```diff
@@ -67,18 +66,18 @@ const css = (
       "&:hover, &:focus-visible": {
         border: `1px solid ${config.colors.danger5}`,
         backgroundColor: config.colors.danger4,
-        color: config.colors.danger0BwInverted,  // danger0BwInverted
+        color: config.colors.danger4BwInverted,
       },
     }),
 
     "call-to-action": () => ({
-      border: `1px solid ${config.colors.happy5}`,
-      color: config.colors.happy4BwInverted, // happy0BwInverted
-      backgroundColor: config.colors.happy4,
+      border: `1px solid ${config.colors.happy7}`,
+      color: config.colors.happy6BwInverted,
+      backgroundColor: config.colors.happy6,
       "&:hover, &:focus-visible": {
-        border: `1px solid ${config.colors.happy6}`,
-        backgroundColor: config.colors.happy5,
-        color: config.colors.happy5BwInverted,  // happy1BwInverted
+        border: `1px solid ${config.colors.happy8}`,
+        backgroundColor: config.colors.happy7,
+        color: config.colors.happy7BwInverted,
```

Comment:
I suggested +1 to each of the colors, i.e. non-hover background `happy7` instead of `happy6`. But I now understood why you didn't take my suggestion 1:1. Because `accent9` and therefore `happy9` is missing from  `config.tsx`! They are only defined in `colors.css`. So I suppose: add those and then +1 all colors used here?

That should also fix the problem you mentioned in your last comment, I think.